Hashids/Hashids.swift

```diff
@@ -52,7 +52,7 @@ public typealias Hashids = Hashids_<UInt32>
 
 public class Hashids_<T where T:Equatable, T:UnsignedIntegerType> : HashidsGenerator
```

Comment:
I haven't taken a look at how Apple does this and it's not in our style guide, but perhaps `<T: Equatable, UnsignedIntegerType>` would be better just because it's shorter (I'm pretty sure that syntax works the same).